AudibleBoulders/AudibleBoulders
# Refactor merge

#### What's this PR do?
Merges the refactor branch into master!

#### What are the important parts of the code?
Everywhere

#### How should this be tested by the reviewer?
From local master, cut a test branch and pull this down. Verify that the app works as expected. Note that the pagination buttons do not yet work.

#### Is any other information necessary to understand this?
#### What are the relevant tickets? (Please add `closes`, `refs`, etc)
#### Screenshots (if appropriate)
